Lib/cmd.py

```diff
@@ -263,7 +263,7 @@ def complete(self, text, state):
             endidx = readline.get_endidx() - stripped
             if begidx>0:
                 cmd, args, foo = self.parseline(line)
-                if cmd == '':
+                if cmd == '' or cmd is None:
```

Comment:
Why not `if not cmd:`?

Reply:
Good point, amended.
Have to add a test still.